packages/drivers/tinylicious-driver/src/insecureTinyliciousUrlResolver.ts

```diff
@@ -93,13 +92,24 @@ export class InsecureTinyliciousUrlResolver implements IUrlResolver {
 }
 
 /**
- * Creates a Routerlicious {@link @fluidframework/core-interfaces#IRequest}.
- * @legacy
- * @alpha
+ * Creates an insecure Tinylicious URL resolver for testing purposes with localhost port 7070.
+ */
+export function createInsecureTinyliciousTestUrlResolver(): IUrlResolver {
+	return new InsecureTinyliciousUrlResolver();
+}
+
+/**
+ * Creates a Tinylicious {@link @fluidframework/core-interfaces#IRequest}.
+ * @internal
  */
 export const createTinyliciousCreateNewRequest = (documentId?: string): IRequest => ({
 	url: documentId ?? "",
 	headers: {
 		[DriverHeader.createNew]: true,
 	},
 });
+
+/**
+ * Creates a Tinylicious {@link @fluidframework/core-interfaces#IRequest} for testing purposes.
+ */
+export const createTinyliciousTestCreateNewRequest = createTinyliciousCreateNewRequest;
```

Comment:
Nit: given that this strictly exists for the export in `testUtils`, would it make sense to just move this declaration to that file?

Reply:
We could, but I think it is better here to make it clear that there are such uses. It would be easy to overlook a test exposure in another file.

Reply:
Perhaps. But it doesn't make clear how it's being exported.

Given that we're generating our own role-ups, I wonder if it would be worth adding a custom tag for test exports?

Reply:
We do know how to add them now. I think we can add that but not enforce it easily. I think that still has value and I'll check it out... assuming this goes in.